src/main/java/com/collicode/propertytracker/service/spec/PaymentsService.java

```diff
@@ -8,4 +8,5 @@ public interface PaymentsService {
   void createPayment(PaymentRequestDTO paymentRequestDTO);
   List<PaymentResponseDTO> fetchPaymentByTenantId(long tenantId);
 
+  Object deletePayment(PaymentRequestDTO paymentId);
```

Comment:
It should hae a return type may be String also while delete it should not take an object 
 example 

`String deletePaymentById(String paymentId);`           but we should not delete payment actually as we need to keep records.

Reply:
so should i not create the crud for deleting payments

Reply:
Yes.